stock_production_lot_active/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Stock Production Lot Active",
     "summary": "Allow to archive/unarchive lots/serial numbers",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
     "development_status": "Beta",
```

Comment:
```suggestion
    "development_status": "Production/Stable",
```
I think it could be set as `Production/Stable`.